src/components/AppointmentConfigModal.vue

```diff
@@ -206,6 +218,12 @@ export default {
 			const uri = this.editing.targetCalendarUri
 			return this.ownSortedCalendars.find(cal => this.calendarUrlToUri(cal.url) === uri)
 		},
+		conflictCalendars() {
+			const freebusyUris = this.editing.freebusyUris ?? []
+			return freebusyUris.map(uri => {
```

Comment:
shouldn't it be `.filter`?

Reply:
We have to map it to calendar objects for the `CalendarPicker`. The config saves uri string and the picker accepts calendar objects.

Reply:
We have to map it to calendar objects for the `CalendarPicker`. The config saves uri string and the picker accepts calendar objects.